### ISSUE TITLE
Do scriptsrc and stylehref actually work?

### ISSUE THREAD

**Issue by dotnetshadow** [nabinked/NToastNotify#140]
Hi there, 

I recently tried overriding the scriptsrcand stylehref options but doesn't matter what I try it seems to always try to load from `/_content/NToastNotify/toastr.js`, even though I set another script path.

I'm using .net 5.0, so I'm not sure if something has changed since then?

When I look at the source:
https://github.com/nabinked/NToastNotify/blob/f5bd2ca994f3b7810a01835b6b0e5341d913d4d3/src/Components/NToastNotifyViewComponent.cs#L30

Seems the component will try to reference a local copy of the script? Does this component copy or download the source if you have something like:  `ScriptSrc = "https://cdnjs.cloudflare.com/ajax/libs/toastr.js/2.1.4/toastr.min.js",` ?

As a test I tried the following and it still rendered using the default script
```
}, new NToastNotifyOption
            {
                DefaultSuccessMessage = "Overwritten default success message",
                ScriptSrc = "xyz",
                StyleHref = "xyz"
```

This is what gets rendered out on the page, you can see the libraryDetails show Abc for scriptSrc but the eventual scripts isn't that one:

```
        <script src=/_content/NToastNotify/toastr.js?7.0.0.0 type="text/javascript"></script>
<script>
    if (nToastNotify) {
        nToastNotify.init({
            firstLoadEvent: 'DOMContentLoaded',
            messages: [{"message":"Same for success message","options":{"title":"Success","type":"success"}},{"message":"Overwritten default success message","options":{"title":"Success","type":"success"}},{"message":"This is an information notification.","options":{"title":"Info","type":"info"}},{"message":"This is a warning notification.","options":{"title":"Warning","type":"warning"}},{"message":"Task could not complete successfully.","options":{"title":"Error","type":"error"}}],
            responseHeaderKey: 'X-NToastNotify-Messages',
            requestHeaderKey: 'X-Requested-With',
            libraryDetails:{"varName":"toastr","scriptSrc":"Abc","styleHref":"abc","options":{"extendedTimeOut":150000,"positionClass":"toast-bottom-center","timeOut":250000,"progressBar":true,"type":"success"}},
            disableAjaxToasts:false
        });
    };
</script>
```

The style tag never gets outputted for some reason.

**Comment by nabinked** [comment]
`ScriptSrc` and `StyleHref` does not change the library js path, libraryJsPath is actually the js file used by this library not the js file for `toastr` and `notyjs`. Setting `ScriptSrc` or  `StyleHref` should be used to load the toastr/noty library from custom location instead of using the default cdn url.
If you look at this line, 
`libraryDetails:{"varName":"toastr","scriptSrc":"Abc","styleHref":"abc","options":` your values are being sent to the library to load script/style from Abc/abc

**Comment by nabinked** [comment]
I just tested on the sample projects and yes they both work. Adding some unit tests.

**Comment by dotnetshadow** [comment]
Thanks for looking into this, I'm a little confused because I did try to load the libraries from a custom source in my case I tried to load them from `https://cdnjs.cloudflare.com/ajax/libs/toastr.js/2.1.4/toastr.min.js`. But I'm not seeing how they are getting injected in the page. For example where should the css styles be loaded? How can I verify this on the page?

**Comment by nabinked** [comment]
Check the network tab, your custom js/css files are loaded after the page load

**Comment by dotnetshadow** [comment]
They aren't loaded at all, that's what I was checking

**Comment by dotnetshadow** [comment]
Looks like it's working in the sample project, so must be something in my project. I'm using asp.net core 5.0 so not sure if that has anything to do with it

**Comment by dotnetshadow** [comment]
I think I found my issue, I was previously loading toastr.js via another script on my _layout page. So maybe there is a check in your script to see if the toastr object exists then you don't load the script? Is that right? I think it's a good check to have well done.

One quirk I found is that I hadn't loaded my css file, but because it found the javascript object it didn't decide to load the css either.
